libkbfs/data_types.go

```diff
@@ -565,6 +565,14 @@ type EntryInfo struct {
 	Ctime int64
 }
 
+// EntryInfoExtended extends the EntryInfo with some locally computed
+// attributes.
+type EntryInfoExtended struct {
```

Comment:
This struct shouldn't be necessary -- everywhere in `libfuse` that calls `fillAttr` should already have access to a `libfuse.Folder` object, which has a copy of the handle already (`h`).  So I don't think it's worth complicating the `KBFSOps` interface with a new struct type.  Of course you'd have to get the UID of the logged-in user (probably with `libkbfs.GetCurrentUsernameIfPossible`) from those places in `libfuse`, but that should be easy enough with a wrapper function.  Maybe `fillAttr` should be moved to be a member function of `libfuse.Folder` or something.